volgen/templates/gfproxy.graph

```diff
@@ -0,0 +1,10 @@
+protocol/server
```

Comment:
The file name. Shouldn't it be `gfproxyd.graph` ?

Reply:
Yes, missed it